nwbinspector/internal_configs/dandi.inspector_config.yaml

```diff
@@ -1,8 +1,8 @@
 CRITICAL:  # All the fields under CRITICAL will be required for dandi validate to pass
   - check_subject_exists
   - check_subject_id_exists
+  - check_subject_sex
+  - check_subject_species
```

Comment:
All the current discussions have focused on requiring `age` and range bounds for it.

Are we sure we want to simultaneously make sec/species required at the same time as well, or take it one at a time? Just wanted to check.

Reply:
Well to me species makes a lot more sense. Experimenters may not know age, but they at least should know species!

Reply:
So I would say yes, let's make that required.